clang/include/clang/Sema/Overload.h

```diff
@@ -979,6 +995,20 @@ class Sema;
       return false;
     }
 
+    bool isPerfectMatch(const ASTContext &Ctx) const {
```

Comment:
`isPerfect` vs `isPerfectMatch` are... confusing.  Work to make these easier to understand together would be appreciated.